Apps/Src/UpdateDisplay.c

```diff
@@ -306,6 +308,40 @@ UpdateDisplayError_t UpdateDisplay_SetAccel(uint8_t percent){
 	return ret;
 }
 
+UpdateDisplayError_t UpdateDisplay_SetForwardPower(uint8_t percent) {
+	static uint8_t lastPercentPower = 0;
+	if(percent == lastPercentPower){
+		return UPDATEDISPLAY_ERR_NO_CHANGE;
+	}
+
+	UpdateDisplayError_t ret = UpdateDisplay_SetComponent(MOTOR, percent);
+	assertUpdateDisplayError(ret);
+	
+	if(ret == UPDATEDISPLAY_ERR_NONE) lastPercentPower = percent;
```

Comment:
out of curiosity, is this if statement actually necessary? Is this not handled by the assert above this?